web/src/components/RunActionsMenu/RunActionsMenu.tsx

```diff
@@ -51,21 +43,11 @@ const RunActionsMenu = ({
               key="view-junit"
               onClick={() => {
                 TestRunAnalyticsService.onLoadJUnitReport();
-                loadJUnit(testId, resultId);
+                onJUnit(testId, resultId);
               }}
             >
               JUnit Results
             </Menu.Item>
-            <Menu.Item
-              data-cy="view-test-definition-button"
-              key="view-test-definition"
-              onClick={() => {
-                TestRunAnalyticsService.onLoadTestDefinition();
-                loadDefinition(ResourceType.Test, testId, testVersion);
-              }}
-            >
-              Test Definition
-            </Menu.Item>
```

Comment:
I think this will remove the Test Definition from this screen too.
<img width="1222" alt="Screenshot 2023-06-19 at 14 57 48" src="https://github.com/kubeshop/tracetest/assets/3879892/0d8f4839-8351-4c0d-a646-7ad7cc385544">

Reply:
Yeah, I don't think that should be a big of a deal, if we want to lead users to the automate tab we can instead have it to be a link to that page, what do you think?

Reply:
@kdhamric what do you think?

Reply:
🤔 I like the idea of a link to the `Automate` tab

Reply:
Can we discuss this during tomorrow's planning? also, can we merge this without that fix so we can keep moving and come back if needed?

Reply:
I am fine with sending people to the automate tab.

Reply:
Ticket [[Deployment Tab] Add Automate Link to the Test Run Menu#2766](https://github.com/kubeshop/tracetest/issues/2766)